pkg/apis/operator/v1alpha1/const.go

```diff
@@ -40,8 +40,11 @@ const (
 	// Hub Params
 	EnableDevconsoleIntegrationParam = "enable-devconsole-integration"
 
-	ApiFieldAlpha  = "alpha"
-	ApiFieldStable = "stable"
+	ApiFieldAlpha         = "alpha"
+	ApiFieldStable        = "stable"
+	FullEmbeddedStatus    = "full"
+	BothEmbeddedStatus    = "both"
+	MinimalEmbeddedStatus = "minimal"
```

Comment:
We should refer to the constant to make sure we are up-to-date when they might change https://github.com/tektoncd/pipeline/blob/8a7b0cfa755038f4cfdcd88c314a72a90bcab1a2/pkg/apis/config/feature_flags.go#L29

Reply:
yep ! updated